spring-cloud-gcp-data-datastore/src/test/java/com/google/cloud/spring/data/datastore/it/TestEntityRepository.java

```diff
@@ -134,6 +135,11 @@
 
 	Optional<TestEntity> findFirstByColor(String color);
 
+	Stream<TestEntity> findStreamByColor(String color);
```

Comment:
I didn't that `findStream...` works?
Would this also work:
```
Stream<TestEntity> findByColor(String color);
```

Reply:
So according to [this post](https://stackoverflow.com/questions/37253571/spring-data-jpa-difference-between-findby-findallby) anything between `find` and `By` is ignored by Spring Data, which is neat when we have two same query with same parameters.
But in this case, `Stream<TestEntity> findByColor(String color);` would also work since the other `findByColor` in line 134 has different signature.

Reply:
Very cool! I didn't know that.